### PR TITLE
Added suggested training to references.

### PR DESCRIPTION
**Pull request by PhilippeTurcotteGov**
* Create new root folder `references`.
* Created new reference document `references/suggested-training.md`.

### CODE REVIEW

**Comment by PhilippeTurcotteGov** [comment]
I suppose that we should probably add a link somewhere in the `README.md` file?

**Comment by s-laugh** [comment]
> 
> 
> I suppose that we should probably add a link somewhere in the `README.md` file?

Probably :), something under "Our Content" maybe?

**Comment by PhilippeTurcotteGov** [comment]
Sure. After Recommendations and Guides?

**Comment by s-laugh** [comment]
Sounds good to me.

File: references/suggested-training.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+* **REST / SOAP APIs**
+* **Relational Databases**
+* **Clean Code**
+* **VS 2015**
+
+## Tool for training
+
+### Dev CoP sessions
+
+We have lots to offer at our monthly [ESDC Development Community - Dev CoP](https://www.gcpedia.gc.ca/wiki/ESDC_Development_Community_-_Dev_CoP).
```
Comment (s-laugh) [inline references/suggested-training.md]: Would be outdated link.
```suggestion
We have lots to offer at our monthly [ESDC Development Community - Dev CoP Sessions](/strategy/dates.md).
```

**Pull request merged**